### DISCUSSION TITLE
What to do when an hard disk is full

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [guydavis/machinaris#580] [Q&A]
Hey guys,

I hope you are all doing great. I was wondering if I should do something when an hard disk is full. Assuming I am plotting on hard disk `A`, when it gets full, what should I do?

Thanks

Francesco

**Comment by FrancescoSaverioZuppichini** [comment]
kindly pinging @guydavis :)

**Comment by k2skaterii** [comment]
In your question, hard disk A - Is that your "fast, plotting drive" or is that your "big and slow plot storage drive"?

**Comment by FrancescoSaverioZuppichini** [comment]
No, both hard disk are slow storage drive :)

**Comment by FrancescoSaverioZuppichini** [comment]
pinging @guydavis

**Comment by guydavis** [comment]
> I hope you are all doing great. I was wondering if I should do something when an hard disk is full. Assuming I am plotting on hard disk `A`, when it gets full, what should I do?

Hi, sorry for the delayed response.  Didn't see this one was left as unanswered.  Alright, you could:

1. Stop Plotman and change the `dst` directory in the [configuration file](https://github.com/guydavis/machinaris/wiki/Plotman#destination-storage) to your new drive (locally mounted) and plot to there.
2. Alternatively, you can use the Plotman [archiving](https://github.com/guydavis/machinaris/wiki/Plotman#archiving) capability to distribute completed plots to a set of local or remote drives, based on which has free space remaining. 

Hope this helps!

**Reply by FrancescoSaverioZuppichini** [reply]
okay so the idea is to:

- have `n` disks to plot to (where `n` is the number of plots you can do at the same time)
- more the `n` plots to other disks to store them.

I am also wondering, what happens to old plots? I am using **flexpool** to plot but I was wondering if I should manually remove the old ones

Thanks a lot!

**Reply by guydavis** [reply]
Please see more details on the [Plotman wiki](https://github.com/ericaltendorf/plotman/wiki/Archiving).  As for deleting old plots, I would only recommend that if you have solo (aka "OG") plots from before Chia introduced pooling.  Any portable (aka "PlotNFT") plots should be usable by Flexpool for farming.

**Reply by FrancescoSaverioZuppichini** [reply]
Thanks a lot for the answer!
